test/occurrences/double_wrapped.t/b.ml

```diff
@@ -0,0 +1,15 @@
+module Y = A
```

Comment:
Missing a test for constructors and class types. Also, there could be a type alias and a value alias.

Reply:
I completely removed support for constructors.
It was broken as it relied on a broken concept of constructor paths. Constructors are resolved differently: they can be several constructors with the same path, they are disambiguated using types. This was not done here. I think support for constructors should be in another PR.

I added a test for class types, but I think support for class types could be removed as well, for being too incomplete. It is I think useless to count class types when nothing is done about classes.

Reply:
It sounds reasonable to remove support for constructors and class types.

Reply:
Done.